master/buildbot/test/unit/test_process_build.py

```diff
@@ -750,6 +789,55 @@ def testGetUrl(self):
         url = yield self.build.getUrl()
         self.assertEqual(url, 'http://localhost:8080/#builders/83/builds/3')
 
+    def test_active_builds_metric(self):
+        """
+        The number of active builds is increased when a build starts
+        and decreased when it finishes.
+        """
+        b = self.build
+
+        controller, step_factory = makeControllableStepFactory()
+        b.setStepFactories([step_factory])
+
+        observer = MetricLogObserver()
+        observer.enable()
+        self.addCleanup(observer.disable)
+
+        def get_active_builds():
+            return observer.asDict()['counters'].get('active_builds', 0)
```

Comment:
mmhh not sure if we want to continue supporting those metrics.
We have the same info in the db, and it is multimaster ready.

I am not sure who are the consumers of those metrics (the web ui, and reporters are not)